### PR TITLE
Add moderator shortcuts to open search and switch between queues.

### PR DESCRIPTION
**Pull request by gerardogalvez**
## What does this PR do?
Addresses part of issue #1025
Adds moderator shortcuts to bring up search (`ctrl+F`) and switch between mod queues (`t`).

## How do I test this PR?
While in the Moderate tab, press `ctrl+F` to bring up Search and press `t` to switch between queues on the selected stream.

**New shortcuts**
screen shot 2017-10-05 at 5 06 28 pm

**Switching between mod queues**

### CODE REVIEW

**Review by kgardnr — APPROVED** [review]

File: client/coral-admin/src/routes/Moderation/components/Moderation.js [hunk 1]
```diff
@@ -36,6 +38,26 @@ class Moderation extends Component {
...
+      key: queue
+    }));
+
+    let activeTabIndex, nextQueueIndex;
+    for (let i = 0; i < menuItems.length; i++) {
+      if (menuItems[i].key === activeTab){
+        activeTabIndex = i;
+        break;
+      }
+    }
```
Comment (alinhle) [inline client/coral-admin/src/routes/Moderation/components/Moderation.js]: What do you think of this?
```js
const activeTabIndex = menuItems.findIndex((item) => item.key === activeTab)
```

Comment (gerardogalvez) [reply client/coral-admin/src/routes/Moderation/components/Moderation.js]: Looks a lot cleaner! I'll get to it as soon as I can. Any other suggestions?

Comment (alinhle) [reply client/coral-admin/src/routes/Moderation/components/Moderation.js]: The rest looks great, thanks for you contribution @gerardogalvez!

**Review by alinhle — CHANGES_REQUESTED** [review]
We'll merge it after the small remark was addressed. This is a great contribution, thx.

File: locales/en.yml [hunk 2]
```diff
@@ -277,6 +277,8 @@ en:
     newest_first: "Newest First"
     navigation: Navigation
     next_comment: "Go to the next comment"
+    toggle_search: "Open search"
```
Comment (okbel) [inline locales/en.yml:280]: Could you add them in spanish too?

```
toggle_search: "Abrir búsqueda"
next_queue: "Próxima cola"
```

**Review by okbel — DISMISSED** [review]
Could you add the translations in spanish too? Thanks for your contribution!

File: locales/en.yml (hunk 2 above)
Comment (gerardogalvez) [reply locales/en.yml:280]: Sure! French and Portuguese are going to be translated by someone else? Or should I attempt to translate?

**Review by alinhle — APPROVED** [review]
Thanks @gerardogalvez. Great stuff!

**Pull request merged**